src/unix/unix_c/unix_madvise.c

```diff
@@ -14,6 +14,16 @@
 
 static int advise_table[] = {
     MADV_NORMAL, MADV_RANDOM, MADV_SEQUENTIAL, MADV_WILLNEED, MADV_DONTNEED,
+#if defined(MADV_MERGEABLE) && defined(MADV_UNMERGEABLE)
```

Comment:
I suggest to define each constant independently, i.e. four `#ifdef`s without the `&&`.

Reply:
@aantron they were introduced together and have no meaning without each other. So they should be presented or both ignored.

Reply:
If that's the case, then Lwt checking for both is redundant, since they will always either both be present or both be absent. On the other hand, if, for any reason, some platform ever has only one flag, then Lwt won't be able to expose the behavior of that platform, because of this check.

I suggest we pass through the platform's own flags as transparently as possible, so that we rely on the platform's logic fully.

Reply:
Done.